guides/micronaut-cloud-secrets-azure/micronaut-cloud-secrets-azure.adoc

```diff
@@ -0,0 +1,194 @@
+    driverClassName: ${JDBC-DRIVER:com.mysql.cj.jdbc.Driver}
+
+flyway:
+  datasources:
+    default:
+      enabled: true
+----
+
+NOTE: The Azure Key Vault secrets naming convention does not allow "_" to be used as a word joiner. Therefore, when adding JDBC secret placeholders, you should use "-" instead (e.q. JDBC-USER, JDBC-PASSWORD).
+
+=== Enable Distributed Configuration
+
+This is accomplished by creating a `bootstrap.yml` file in the `resources` directory and adding the following:
+
+[,yml]
+.src/main/resources/bootstrap.yml
+----
+micronaut:
+  application:
+    name: micronautguide # <1>
+  config-client:
+    enabled: true # <2>
+
+azure:
+  credential:
+    client-secret:
+      client-id: ${AZURE_CLIENT_ID} # <3>
+      tenant-id: ${AZURE_TENANT_ID} # <4>
+      secret: ${AZURE_SECRET_VALUE} # <5>
+  key-vault:
+    vault-url: ${AZURE_VAULT_URI} # <6>
+----
+<1> The application name entry is moved from `src/main/resources/application.yml` for resolving distributed configurations properties
+<2> Flag to enable the distributed configurations
+<3> The client ID of the trusted application that you will register in Azure Portal
+<4> The tenant ID
+<5> The secret value of the registered application
+<6> The Azure Key Vault resource URI
+
+NOTE: More details on how to obtain these values are provided in the Azure Portal section below.
+
+https://docs.micronaut.io/latest/guide/#bootstrap[Click here to get more details on distributed configuration]
+
+=== Clean up of the Application Configuration
+
+Remove the redundant application name entry from the `application.yml`.
+
+[,diff]
+.src/main/resources/application.yml
+----
+- micronaut:
+-  application:
+-    name: micronautguide
+
+datasources:
+  default:
+    username: ${JDBC-USER}
+    password: ${JDBC-PASSWORD}
+    dialect: MYSQL
+    driverClassName: ${JDBC-DRIVER:com.mysql.cj.jdbc.Driver}
+
+flyway:
+  datasources:
+    default:
+      enabled: true
+----
+
+== Store JDBC secrets in Azure Key Vault using Azure Portal
+
+Create Azure Key Vault service resource.
+{empty} +
+
+image::azure-key-vault/azure-vault-create-vault-resouce.png[]
+{empty} +
+
+When Key Vault service deployment is ready, go to the resource in order to create secrets. Prior to clicking on the Secrets option in the left menu list, copy the Vault URI from the main screen section. You will need to set it as an environment variable in order to integrate Azure Key Vault into your application.
+{empty} +
+
+image::azure-key-vault/azure-vault-obtain-vault-uri.png[]
+{empty} +
+
+Create new secrets for JDBC user and password. Click on the Secrets option in the left menu, then click on Generate/Import.
+{empty} +
+
+image::azure-key-vault/azure-vault-create-secret-jdbc-user.png[]
+{empty} +
+
+image::azure-key-vault/azure-vault-create-secret-jdbc-password.png[]
+{empty} +
+
+Go to App registration in order to register your application with the Microsoft identity platform. After registering your application,
```

Comment:
```suggestion
Go to App registration to register your application with the Microsoft identity platform. After registering your application,
```

Reply:
Covered in https://github.com/micronaut-projects/micronaut-guides/pull/1099/commits/27ffa927a580938d2c8edd451d078f4c25369a2f.